### DISCUSSION TITLE
Oracle Fatal Background Processes | Tanel Poder Consulting

### DISCUSSION THREAD

**Discussion by giscus** [tanelpoder/blog-comments#48] [Announcements]
# Oracle Fatal Background Processes | Tanel Poder Consulting

When talking about Oracle background processes, there&rsquo;s a term/qualifier &ldquo;fatal&rdquo; background process. This means that when one of these background processes crashes, then whoever detects the process disappearance (PMON or LGWR or CLMN possibly), will shut down the instance as it cannot function normally anymore.
Not all background process crashes take down the whole instance, for example processes like J000 and P000 are technically background processes (daemons disconnected from network), but their crash won&rsquo;t take down the instance. - Linux, Oracle, SQL performance tuning and troubleshooting - consulting & training.

https://tanelpoder.com/posts/oracle-fatal-background-processes/

**Comment by jeanremacle** [comment]
Hello,

I was wondering about the attribute "instance_critical" found in the ATTRIBUTES column, which is available in (G)V$PROCESS and (G)VBGPROCESS.
Do you think it is linked to the same source as explained in your post?

By the way, while writing this message, I noticed it's only available in Oracle 23c. 
There is also a new FLAG column in (G)V$BGPROCESS, but I am not sure what its representation is.

Regards,
Jean

**Reply by tanelpoder** [reply]
Yep that's often the case: some feature is added and new columns show up in X$ tables and a release or two later, these columns get exposed to V$ too.
